.pipelines/templates/rp-dev/rp-dev-vars.yml

```diff
@@ -0,0 +1,8 @@
+variables:
+  AZURE_UNIQUE_PREFIX: zzz # dummy value
+  NO_CACHE: false
+  AZURE_EXTENSION_DEV_SOURCES: "$(pwd)/python"
+  ARO_INSTALL_VIA_HIVE: true
+  ARO_ADOPT_BY_HIVE: true
+  ARO_SKIP_PKI_TESTS: true
```

Comment:
I think we can eliminate this, since it's only used for E2E test suite runs, which don't happen as part of deployment, correct?

Reply:
I think that's accurate. Would it make sense to discard it as well in https://github.com/Azure/ARO-RP/blob/master/env.example#L6?